crates/source_gen/src/lexer/mod.rs

```diff
@@ -85,4 +85,14 @@ mod tests {
     fn lex_quoted_ident() {
         check(r#""读文👩🏼‍🔬""#, T![quoted_ident]);
     }
+
+    #[test]
+    fn test_integer() {
+        check(r#"69"#, T![int_literal]);
+    }
+
+    #[test]
+    fn test_decimal() {
+        check(r#"420.69"#, T![decimal_literal]);
```

Comment:
Just tested it out, and seems SQL allows `420.`, as well as `.69` (at least both in Oracle and PostgreSQL).